### PR TITLE
Clarify Desktop update behavior

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Clarify how the auto updates feature works in Docker Desktop

### CODE REVIEW

**Comment by usha-mandya** [comment]
@dieucao @mat007 PTAL - https://deploy-preview-12801--docsdocker.netlify.app/docker-for-mac/install/#automatic-updates

File: _includes/desktop-update.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!-- This text will be included in the Mac and Windows Install topic -->
+
+Starting with Docker Desktop 3.0.0, updates to Docker Desktop will be available automatically as delta updates from the previous version. When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can choose when to start the download and installation process.
+
+> To encourage developers to stay up to date, Docker Desktop displays a reminder two weeks after an update becomes available. You can dismiss this daily reminder using the **Snooze** option.
```
Comment (mat007) [inline _includes/desktop-update.md:5]: An option sounds a bit like a setting to me, maybe some users will start asking where to find this option in the settings?

File: docker-for-mac/install.md [hunk 2]
```diff
@@ -91,21 +91,10 @@ and select **Learn**.
...
-When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can start downloading the update in the background whenever it is convenient for you.
-
-After downloading the update, all you need to do is to click **Update and restart** from the Docker menu. This installs the latest update and restarts Docker Desktop for the changes to take effect.
-
-## Uninstall Docker Desktop
-
-To uninstall Docker Desktop from your Mac:
-
-1. From the Docker menu, select **Troubleshoot** and then select **Uninstall**.
-2. Click **Uninstall** to confirm your selection.
```
Comment (mat007) [inline docker-for-mac/install.md:105]: I don’t think this was meant to be removed, was it?

Comment (usha-mandya) [reply docker-for-mac/install.md]: Good catch. Fixed it 😅

File: _includes/desktop-update.md (hunk 1 above)
Comment (usha-mandya) [reply _includes/desktop-update.md]: @mat007 Updated it to 'clicking' Snooze. Also, there's a screenshot that displays this option. 

image

**Review by mat007 — APPROVED** [review]
Looks good to me!

**Comment by dieucao** [comment]
@usha-mandya Suggest changing the title of the section from Automatic Updates because we are no longer automatically updating. Maybe Delta Updates?

**Review by StefanScherer — APPROVED** [review]
LGTM

**Pull request merged**